client/src/screens/Notification.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import { useNavigation } from '@react-navigation/native'
+import { FlatList, Avatar } from 'native-base'
+
+import ListItemBarColored from '../elements/ListItemBarColored'
+
+export default () => {
+  const { user } = useSelector((state) => state.auth)
+  const notifications = user.notifications
+  const navigation = useNavigation()
+
+  const onPress = (item) => navigation.navigate('MyProductsTab', { screen: 'MyProduct', params: { id: item._id } })
```

Comment:
item._id is notidfication id, not product id. Need to store product id in notification object and pass it to navigate params